boards/FSM/Src/Io/Io_Brake.c

```diff
@@ -3,8 +3,8 @@
 #include "Io_Adc.h"
 
 // TODO set these values
```

Comment:
Can we remove this now?

Reply:
These are still guesses, especially for the pedal